### PR TITLE
Solve findVolumeOfACuboid challenge using typescript

### LINKED ISSUE #87: Write a TypeScript program to find volume of a cuboid
### Description

Write a TypeScript program to find volume of a cuboid

Volume of a cuboid = `Length` × `Width` × `Height`

```
Length : 2
Width  : 3
Height : 4

Volume : 24
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACuboid.ts` file
- Add `FindVolumeOfACuboid.ts` file in `find-volume-of-a-cuboid` folder

</details>

### LINKED ISSUE #104: Write a TypeScript program to find volume of a cuboid
### Description

Write a TypeScript program to find volume of a cuboid

Volume of a cuboid = `Length` × `Width` × `Height`

```
Length : 2
Width  : 3
Height : 4

Volume : 24
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACuboid.ts` file
- Add `FindVolumeOfACuboid.ts` file in `find-volume-of-a-cuboid` folder

</details>

### PR DESCRIPTION
**Pull request by muhammadjufry**
Closes codinasion/codinasion#87 

### What's being changed:
Write typescript program to find the volume of a cuboid

### Programming Language:
TypeScript

### CODE REVIEW

**Comment by harshraj8843** [comment]
Hey @MuhammadJufri 

Please link the related issue

> Closes #ISSUE_NUMBER

For example `Closes codinasion/codinasion#2`

**Comment by MuhammadJufri** [comment]
Ok

**Comment by MuhammadJufri** [comment]
I've just updated!

File: program/find-volume-of-a-cuboid/findVolumeOfACuboid.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function findVolumeOfACuboid(length: number, width: number, height: number) {
+  return length * width * height;
+}
+
+console.log(2,3,4);
```
Comment (victoriacheng15) [inline program/find-volume-of-a-cuboid/findVolumeOfACuboid.ts:5]: This should be `console.log(findVolumneOfACuboid(2, 3, 4))`. Otherwise, it looks good

**Review by victoriacheng15 — CHANGES_REQUESTED** [review]
See comment below

File: program/find-volume-of-a-cuboid/findVolumeOfACuboid.ts (hunk 1 above)
Comment (MuhammadJufri) [reply program/find-volume-of-a-cuboid/findVolumeOfACuboid.ts:5]: Ooh Sorry

Comment (MuhammadJufri) [reply program/find-volume-of-a-cuboid/findVolumeOfACuboid.ts]: I've just updated it!

Comment (victoriacheng15) [reply program/find-volume-of-a-cuboid/findVolumeOfACuboid.ts]: No problem and thank you for updating! 👍

**Review by victoriacheng15 — APPROVED** [review]

**Review by SpirosArk — APPROVED** [review]

**Pull request merged**